goty-server/src/main/kotlin/com/aleinin/goty/game/GameSearchConfiguration.kt

```diff
@@ -0,0 +1,42 @@
+package com.aleinin.goty.game
+
+import com.api.igdb.request.IGDBWrapper
+import com.api.igdb.request.IGDBWrapper.setCredentials
+import com.api.igdb.request.TwitchAuthenticator.requestTwitchToken
+import org.slf4j.LoggerFactory
+import org.springframework.context.ApplicationListener
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Profile
+import org.springframework.context.event.ContextRefreshedEvent
+import javax.validation.Valid
+
+private val logger = LoggerFactory.getLogger(DefaultUnconfiguredSearchConfiguration::class.java)
```

Comment:
should this not be moved down? It seems specific to DefaultUnconfiguredSearchConfiguration. Perhaps it should be a companion (static) object so it doesn't pollute the IgdbSearchConfiguration namespace?

Reply:
Good point. Yes, it probably should be isolated to just `DefaultUnconfiguredSearchConfiguration`.

Reply:
Addressed by 5c8085c2c848fee390b56d58c4de07385bd693ec